test/periphery/PixCashierUpgradeable.test.ts

```diff
@@ -50,25 +53,30 @@ describe("Contract 'PixCashierUpgradeable'", async () => {
 
     it("Is reverted if caller is not whitelisted", async () => {
       await proveTx(pixCashier.setWhitelistEnabled(true));
-      await expect(pixCashier.cashIn(user.address, tokenAmount))
+      await expect(pixCashier.cashIn(user.address, tokenAmount, TRANSACTION_ID))
```

Comment:
Codacy has a fix for the issue: [Replace `pixCashier.cashIn(user.address,·tokenAmount,·TRANSACTION_ID))` with `⏎········pixCashier.cashIn(user.address,·tokenAmount,·TRANSACTION_ID)`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9795032)
```suggestion
      await expect(
```